en/services/illuminator.md

```diff
@@ -0,0 +1,154 @@
+
+<!-- Mermaid graph:
+sequenceDiagram;
+    participant Drone
+    participant Illuminator
+    Drone->>Illuminator: MAV_CMD_ILLUMINATOR_ON_OFF (param1 = 1)
+    Drone->>Drone: Start timeout
+    Illuminator->>Drone: MAV_RESULT_ACCEPTED
+-->
+
+[![](https://mermaid.ink/img/pako:eNplkMFqwzAMhl_F6LRCd-jVZYWQpBBImtGkOxmMSLTNLLYzVz6M0nef2zLWMV0kpE__j3SCwY8EEo70GckNVBh8C2jXyokUMwY2g5nRsSiCd_S_XU1TtMYh-3AbXrnHzeZuIEWTvei8KXRV14em2mV9u9ftTrfbrXhIamhX4kmsFn8VrlmKjpOdYGPJR74Rd9q_3MVjX3aHutdZnpfPfVnAEiwFi2ZMJ54uuwr4nSwpkKkcMXwoUO6cOIzsuy83gOQQaQlxHpF_3gHyFacjnb8BIBNkuQ?type=png)](https://mermaid.live/edit#pako:eNplkMFqwzAMhl_F6LRCd-jVZYWQpBBImtGkOxmMSLTNLLYzVz6M0nef2zLWMV0kpE__j3SCwY8EEo70GckNVBh8C2jXyokUMwY2g5nRsSiCd_S_XU1TtMYh-3AbXrnHzeZuIEWTvei8KXRV14em2mV9u9ftTrfbrXhIamhX4kmsFn8VrlmKjpOdYGPJR74Rd9q_3MVjX3aHutdZnpfPfVnAEiwFi2ZMJ54uuwr4nSwpkKkcMXwoUO6cOIzsuy83gOQQaQlxHpF_3gHyFacjnb8BIBNkuQ)
+
+<!-- This blank line provides visual separation in the document -->
+<!-- This blank line provides visual separation in the document -->
+
+#### CONFIGURE
+
+The [MAV_CMD_DO_ILLUMINATOR_CONFIGURE](#MAV_CMD_DO_ILLUMINATOR_CONFIGURE) command controls the illuminator's settings. This will adjust how the illuminator behaves when enabled. The operation follows the normal [Command Protocol](https://github.com/mavlink/mavlink-devguide/blob/master/en/services/command.md) rules for command/acknowledgment. The four parameters for this command are: Mode, Brightness, Strobe Period, Strobe Duty.
+
+Illuminators can be set in different modes which can change the behavior of the illuminator (described in a separate section, [Modes](#MODES)). The brightness can be set via "Brightness" as a percentage value (0-100%). Illuminators may also have the functionality to strobe the light source. This behavior is configured via "Strobe Period" and "Strobe Duty". These parameters can be set to 0 when not used. "Strobe Period" is in seconds and "Strobe Duty" is a percentage value (indicating the % of time in the "Strobe Period" the illuminator is enabled).
+
+<!-- This blank line provides visual separation in the document -->
+<!-- This blank line provides visual separation in the document -->
+
+##### MODES
+
+Illuminator modes can help the drone configure the interaction behavior of the illuminator.
+
+If the mode is unknown, the mode parameter value will be set to 0.
+
+The mode value as 1 indicates `ILLUMINATOR_MODE_INTERNAL_CONTROL` where the illuminator behavior is controlled by [MAV_CMD_DO_ILLUMINATOR_CONFIGURE](#MAV_CMD_DO_ILLUMINATOR_CONFIGURE) settings.
+
+When the value is set to 2, it indicates that the illuminator mode is `ILLUMINATOR_MODE_EXTERNAL_SYNC`. This mode is for instances where the illuminator behavior is controlled by external factors: e.g. an external hardware signal.
+
+<!-- This blank line provides visual separation in the document -->
+<!-- This blank line provides visual separation in the document -->

```

Comment:
I wouldn't worry too much about visual separation. That should be fixed in the theme but we'll live with it for now. 
```suggestion
```

Reply:
Ditto for the cases below